.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+on:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    name: build dependencies & create artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
+      - name: Build JS dependencies
+        uses: PrestaShopCorp/github-action-build-js/12@v1.0
+        with:
+          cmd: yarn
+          path: ./_dev
+      - name: Install composer dependencies
+        run: composer install --no-dev -o
+      - name: Clean-up project
+        uses: PrestaShopCorp/github-action-clean-before-deploy@v1.0
+      - name: Create & upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: ps_checkout
+          path: ../
+      - name: Publish to GitHub Release
+        if: github.event_name == 'release' && github.event.action == 'published'
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.release_info.outputs.upload_url }}
+          asset_path: ./build/ps_checkout.zip
+          asset_name: ps_checkout.zip
```

Comment:
It would be great to find a way to make this script generic, so we just have to copy-paste the workflow on the next modules.

A solution has been suggested about this, but I haven't checked yet: https://stackoverflow.com/a/58474340

Reply:
Tried on my fork which has a custom name: 
![image](https://user-images.githubusercontent.com/6768917/74734904-d8b73200-524f-11ea-909e-200ce361874d.png)